app/src/main/res/values/strings.xml

```diff
@@ -390,7 +390,6 @@
   <string name="preference_author_name_toggle">Use custom author name</string>
   <string name="preference_author_name_toggle_summary">Use a custom author name instead of your username while uploading photos</string>
   <string name="preference_author_name">Custom author name</string>
-  <string name="preference_author_name_summary">The custom author name to use instead of your username in uploads</string>
```

Comment:
not removed for every other locale, run a lint inspection and they should show up as unused resources to delete

Reply:
@macgills I think the **translate-wiki** automatically removes them from other locals

Reply:
Pinging @misaochan for getting more information on this.

Reply:
I believe you're right. Translate wiki handles this by itself. Ref: [FAQ](https://translatewiki.net/wiki/FAQ#Do_I_have_to_do_anything_special_when_deleting_messages?)

Reply:
Coolio